packages/nimble-components/src/text-area/styles.ts

```diff
@@ -10,13 +10,19 @@ import {
     controlLabelFont,
     controlLabelFontColor,
     bodyFont,
-    controlLabelDisabledFontColor
+    controlLabelDisabledFontColor,
```

Comment:
width seems to behave weird with error icon

Reply:
It turns out we already had some issues with resizing the control after setting `width` and/or `height` on it. Specifically, the inner `textarea` resizes, but the host element keeps the same fixed size, so you can get controls overlapping:
![image](https://user-images.githubusercontent.com/7282195/229212881-8268f289-1461-4308-9fc1-c4ef7b86fb7d.png)
This issue was somewhat complicated further by the introduction of a container `div` around the inner `textarea` to support the error state UI.

I tried various `display` modes in combination with different `height`/`width` values for the container `div` as well as the host element. I was unable to find a way to get the `nimble-text-area` to ignore the specified `height`/`width` in favor of the `height`/`width` set on the inner `textarea` while manually sizing. The closest I came was by having my `ResizeObserver` also observe the inner `textarea` and clear the host's `height`/`width` when the `textarea`'s dimensions changed. That doesn't seem like what we want to be doing.

Instead, I added a note in our documentation of the `resize` attribute that users should not set a fixed `height`/`width` when making their text area resizeable. I instead suggest they use `rows`/`cols` to set an initial size, which works as expected.

Reply:
Sounds like a reasonable trade-off to me 👍